apps/io-services-cms-webapp/src/utils/subscription.ts

```diff
@@ -24,6 +29,23 @@ type ErrorResponses =
 const isManageKey = (ownerSubscriptionId: NonEmptyString) =>
   ownerSubscriptionId.startsWith(MANAGE_APIKEY_PREFIX);
 
+const areUserAndOwnerEquals = (
+  ownerId: NonEmptyString,
+  userId: NonEmptyString
+) => ownerId === userId;
+
+const extractOwnerId = (
+  fullPath?: string
+): TaskEither<IResponseErrorNotFound, NonEmptyString> =>
```

Comment:
it looks like this method is synchronous, why do we need to return TE?

Reply:
009e4dc